flax/configurations.py

```diff
@@ -120,3 +119,9 @@ def __exit__(self, *exc):
     name='return_frozendict',
     default=True,
     help=('Whether to return FrozenDicts when calling init or apply.'))
+
+flax_fix_rng = define_bool_state(
+    name ='fix_rng',
```

Comment:
could we give the flag a more descriptive name?  `fix_rng_unique_derivation` or something?

Reply:
updated to `fix_rng_separator`